code/game/objects/items/theft_items.dm

```diff
@@ -202,13 +247,52 @@
 		if(ismob(loc))
 			to_chat(loc, "<span class='warning'>[src] is permanently sealed, [sliver] is safely contained.</span>")
 
+/obj/item/nuke_core_container/supermatter/proc/unload(obj/item/retractor/supermatter/I, mob/user)
+	if(!istype(I) || I.sliver)
+		return
+	sliver.forceMove(I)
+	I.sliver = sliver
+	sliver = null
+	I.icon_state = "supermatter_tongs"
+	I.item_state = "supermatter_tongs"
+	I.icon_state = "supermatter_tongs_loaded"
+	I.item_state = "supermatter_tongs_loaded"
```

Comment:
Ehm... setting the `icon_state` and `item_state` double here?

Reply:
Oops